src/handlers/mentionCommands.ts

```diff
@@ -47,6 +47,8 @@ async function handleCommand(message: Message<true>, document: GuildDocument, ex
     command.debugLevel === DebugCommandLevel.Admin && !config.admins.includes(message.author.id)
   ) return reply("⛔ This command is not available.", message, existingReply).then(newReply => [message, newReply]);
 
+  if (command.requireSelectedCountingChannel && document.channels.size === 0) return reply("💥 No counting channel is set up in this server! Create a new one by using `/channels new` or link an existing one with `/channels link`.", message, existingReply).then(newReply => [message, newReply]);
```

Comment:
maybe include docs link here as well for consistency?

Reply:
I thought about it but didn't include it here because of markdown 🤷‍♂️

if others say the same thing then we can add it as a plain url